content/applications/marketing/events.rst

```diff
@@ -4,12 +4,172 @@
+
+To quickly add a new event to a pipeline, click the :guilabel:`➕ (plus)` icon at the top of the
+stage to which the event should be added to reveal a blank Kanban card to fill out.
+
+.. image:: events/blank-kanban-card.png
+   :align: center
+   :alt: Typical blank kanban card to fill out in the Odoo Events application.
+
+In this blank Kanban card, enter the name of :guilabel:`Event`, along with the start and end
+:guilabel:`Date` and time.
+
+Then, either click :guilabel:`Add` to add it to the stage and edit it later, or click
+:guilabel:`Edit` to add the event to stage and edit its configurations on a separate page.
+
+Each event card can be dragged-and-dropped into any stage on the Kanban pipeline, providing easy
+organizational access.
+
+Settings
+========
+
+To access the event settings and feature options in Odoo *Events*, navigate to
+:menuselection:`Events app --> Configuration --> Settings`. From here, tick the checkboxes beside
+the desired settings and/or features, and click :guilabel:`Save` to activate them.
+
+Events section
+--------------
+
+In the :guilabel:`Events` section of the :guilabel:`Settings` page, there are selectable features
+that can be enabled to add various elements to events created with the Odoo *Events* application.
+
+.. image:: events/settings-events-section.png
+   :align: center
+   :alt: The Events section on the Odoo Events settings page in the Odoo Events application.
+
+The :guilabel:`Schedule & Tracks` feature allows users to manage and publish a schedule with tracks
+for events. *Tracks* is a catch-all term that refers to talks, lectures, demostrations,
+presentations, and other similar elements that users may choose to include as part of an event.
+
+When the :guilabel:`Schedule & Tracks` feature is enabled, two additional fields appear beneath it:
+:guilabel:`Live Broadcast` and :guilabel:`Event Gamification`.
+
+The :guilabel:`Live Broadcast` feature lets users air tracks online, via a *YouTube* integration.
+
+The :guilabel:`Event Gamification` feature lets users share a quiz after any event track, in order
+for attendees to gauge how much they learned from the track they just saw/heard. Companies can also
+benefit from this feature, in that the subsequent responses and results of the quizzes can help
+determine where a company's strengths and weaknesses are, when it comes to their presentations.
+
+Next, is the :guilabel:`Online Exhibitors` feature. This feature allows uers to display sponsors and
+exhibitors on event pages, which can serve as a valuable incentive to encourage partners and
+businesses to participate in the event.
+
+The :guilabel:`Jitsi Server Domain` field represents an external conferencing service that is
+integrated with Odoo. It is what is used to create and host virtual conferences, community rooms,
+and other similar elements for events.
+
+.. note::
+   The :guilabel:`Jitsi Server Domain` field is populated, by default.
```

Comment:
Can remove this **note** I think, unless we want to add more specific information about what the default server address is/means, how to change it, etc.